base/asyncevent.jl

```diff
@@ -90,6 +90,8 @@ mutable struct Timer
     cond::ThreadSynchronizer
     @atomic isopen::Bool
     @atomic set::Bool
+    timeout::Real
+    interval::Real
```

Comment:
It turns out libuv has an accessor for this, `uv_timer_get_repeat` https://docs.libuv.org/en/v1.x/timer.html#c.uv_timer_get_repeat, so we can use that instead of storing it.

Reply:
I tried this but it won't work if the timer has been closed
```
julia> t = Timer(1, interval = 0.1)
Timer (open, timeout: 1.0 s, interval: 0.1 s) @0x000000010cada9b0

julia> close(t)

julia> t

[91257] signal 11 (2): Segmentation fault: 11
in expression starting at REPL[4]:1
uv_timer_get_repeat at /workspace/srcdir/libuv/src/timer.c:120
getproperty at ./asyncevent.jl:126 [inlined]
show at ./asyncevent.jl:135
show at ./multimedia.jl:47
```

So I've stored both as their ms values.